packages/logo/src/ProductLogos.tsx

```diff
@@ -0,0 +1,463 @@
+import React from 'react';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { HTMLElementProps } from '@leafygreen-ui/lib';
+
+type ProductLogoProps = HTMLElementProps<'svg'> & {
+  knockout?: boolean;
+  size?: number;
+};
+
+export function CloudManagerLogo({
+  size = 18,
+  knockout = false,
+  className,
+  ...rest
+}: ProductLogoProps) {
+  return (
+    <svg
+      {...rest}
+      width={`${size}px`}
+      height={`${size}px`}
+      viewBox={`0 0 18 18`}
```

Comment:
This doesn't need to be wrapped in curly braces: `viewBox='0 0 18 18'`

Reply:
👍